cli/src/main/java/org/jboss/as/cli/impl/CliLauncher.java

```diff
@@ -165,6 +165,8 @@ public static void main(String[] args) throws Exception {
                     ctxBuilder.setEchoCommand(true);
                 } else if (arg.equals("--output-json")) {
                     ctxBuilder.setOutputJSON(true);
+                } else if (arg.equals("--color-output")) {
+                    ctxBuilder.setColorOutput(true);
```

Comment:
Shouldn't the color output be enabled by default? See a [comment](https://github.com/wildfly/wildfly-proposals/pull/33/#discussion_r174834029).

Reply:
Actually, if we want the "bang" effect on users, we should enable it by default. If we got too much feedbacks that it is disruptive, we can default to false.

Reply:
I am playing with this feature and the colored prompt seems to me disruptive a bit. IMO we should use colors for highlighting importing information like required parameters or error logs.

In following screenshot I like the first line. It highlights that the CLI client is disconnected. However the last two lines seem disruptive to me. There are too many colors and they highlight prompt which is not important in this case. I would rather de-emphasize it by gray color, but default (black) would be fine as well. Wdyt?

![screenshot](https://user-images.githubusercontent.com/2317374/39423557-be997d90-4c72-11e8-8c27-68045c94d0f5.png)